testing/py3-maxminddb/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor: Kevin Daudt <kdaudt@alpinelinux.org>
+# Maintainer: Kevin Daudt <kdaudt@alpinelinux.org>
+pkgname=py3-maxminddb
```

Comment:
how that related to upgrade?

Reply:
I forgot to commit one package, which I just pushed.

sopel -> py3-geoip2 -> py3-maxminddb